styleguide/src/utils/elapsedDuration.ts

```diff
@@ -0,0 +1,26 @@
+const MS_IN_SECOND = 1000
+const SECONDS_IN_MINUTE = 60
+const SECONDS_IN_HOUR = 3600
+
+export function elapsedDuration(createdAt: string, finishedAt: string): string {
+  if (!createdAt && !finishedAt) {
```

Comment:
Should we make this more explicit? Something like
```suggestion
  if (createdAt === '' && finishedAt === '') {
```